src/pudl_archiver/orchestrator.py

```diff
@@ -160,41 +166,8 @@ async def _initialize(self):
 
         # TODO (daz): stop using self.deposition_files, use the files lists on the depositions
         # Map file name to file metadata for all files in deposition
-        self.deposition_files = await self._remote_fileinfo(self.new_deposition)
-
-    # TODO (daz): inline this.
-    async def _remote_fileinfo(self, deposition: Deposition):
-        """Return info on all files contained in deposition.
-
-        Args:
-            deposition: Deposition for which to return file info.
-
-        Returns:
-            Dictionary mapping filenames to DepositionFile metadata objects.
-        """
-        return {f.filename: f for f in deposition.files}
-
-    # TODO (daz): inline this.
-    async def _create_deposition(self, data_source_id: str) -> Deposition:
-        """Create a Zenodo deposition resource.
-
-        This should only be called once for a given data source.  The deposition will be
-        prepared in draft form, so that files can be added prior to publication.
-
-        Args:
-            data_source_id: Data source ID that will be used to generate zenodo metadata
-            from data source metadata.
-
-        Returns:
-            Deposition object, per
-            https://developers.zenodo.org/?python#depositions
-        """
-        metadata = DepositionMetadata.from_data_source(data_source_id)
-        if not metadata.keywords:
-            raise AssertionError(
-                "New dataset is missing keywords and cannot be archived."
-            )
-        return await self.depositor.create_deposition(metadata)
+        self.deposition_files = {f.filename: f for f in self.new_deposition.files}
```

Comment:
We could probably start passing the actual deposition objects around since we want them to be *run*-scoped, not *orchestrator*-scoped. But that seems out of scope for this PR.

Reply:
Yeah this has always felt like probably an unnecessary/error prone bit of state to maintain